iothub_client/tests/common_device_method_e2e/iothubclient_common_device_method_e2e.c

```diff
@@ -208,20 +208,68 @@ static int DeviceMethodWithUploadCallback(const char* method_name, const unsigne
 
 void test_invoke_device_method(const char* deviceId, const char* moduleId, const char *payload)
 {
-    int responseStatus;
-    unsigned char* responsePayload;
-    size_t responsePayloadSize;
+    int responseStatus = METHOD_RESPONSE_ERROR;
+    unsigned char* responsePayload = NULL;
+    size_t responsePayloadSize = 0;
+    IOTHUB_DEVICE_METHOD_RESULT invokeResult = IOTHUB_DEVICE_METHOD_ERROR;
 
     if (moduleId != NULL)
     {
-        LogInfo("IoTHubDeviceMethod_InvokeModule deviceId='%s', moduleId='%s'", deviceId, moduleId);
-        IOTHUB_DEVICE_METHOD_RESULT invokeResult = IoTHubDeviceMethod_InvokeModule(serviceClientDeviceMethodHandle, deviceId, moduleId, METHOD_NAME, payload, TIMEOUT, &responseStatus, &responsePayload, &responsePayloadSize);
+        for (int tryCounter = 0; tryCounter < TEST_METHOD_INVOKE_MAX_RETRIES; tryCounter++)
+        {
+            LogInfo("(Try %d) IoTHubDeviceMethod_InvokeModule deviceId='%s', moduleId='%s'", tryCounter + 1, deviceId, moduleId);
```

Comment:
too verbose?

Reply:
I'll remove these and leave only the error reports in place.